assets/emojis/index.ts

```diff
@@ -31,5 +33,21 @@ const localeEmojis = {
     es: esEmojis,
 } as const;
 
-export {emojiNameTable, emojiCodeTableWithSkinTones, localeEmojis};
-export {skinTones, categoryFrequentlyUsed, default} from './common';
+// On windows, flag emojis are not supported
+const emojisForOperatingSystem =
+    getOperatingSystem() === CONST.OS.WINDOWS
+        ? emojis.slice(
+              0,
+              emojis.findIndex((emoji) => {
+                  if (!('header' in emoji)) {
+                      return;
+                  }
+
+                  return emoji.header && emoji.code === 'flags';
+              }),
+          )
+        : emojis;
```

Comment:
@TMisiukiewicz This was implemented during the time you have worked on this PR but we have added the support for flag emojis on Windows in this issues and pr https://github.com/Expensify/App/issues/31717

There was this issue raised here https://github.com/Expensify/App/issues/33593, if you could comment on it so I can assign you and you could make a follow up to remove this Windows specific code. thanks!